test/integration-tests/gulp/runner.ts

```diff
@@ -29,6 +29,30 @@ export class GulpIntegrationTests {
       });
    }
 
+   @AsyncTest()
+   @Timeout(10000)
+   public toThrowExpectations() {
+
+      const result = child
+        .exec("gulp test-to-throw-expectations --gulpfile \"./test/integration-tests/gulp/gulpfile.js\" --cwd ./");
+
+      let consoleOutput = "";
+
+      result.stdout.on("data", (data) => consoleOutput += data);
+      result.stderr.on("data", (data) => consoleOutput += data);
+
+      let expectedOutput = FileSystem
```

Comment:
Because it's text data. Making it JSON would require additional wrapping and make the file less readable. I wanted to keep the text file as near to the readable output as possible